CondCore/SiStripPlugins/test/BuildFile.xml

```diff
@@ -1,4 +1,4 @@
-<use name="CommonTools/TrackerMap"/>
+<use name="CommonTools/TrackerMap"/> 
```

Comment:
Why add space?

Reply:
Excuse me, that was a mistake. Should I fix it now or do you first want to run the tests? Or maybe abort?